src/material/schematics/ng-generate/mdc-migration/rules/project-tsconfig-paths.ts

```diff
@@ -0,0 +1,103 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {json, normalize, virtualFs, workspaces} from '@angular-devkit/core';
+import {Tree} from '@angular-devkit/schematics';
+
+/**
+ * Gets all tsconfig paths from a CLI project by reading the workspace configuration
+ * and looking for common tsconfig locations.
+ */
+export async function getProjectTsConfigPaths(
```

Comment:
I feel like we had a util for this already in cdk/schematics?

Reply:
Ok I'll look for that... This one is from some framework code since I didn't realize we had it